### DISCUSSION TITLE
Smoothness slider missing, how to re-enable or is it even possible or hello?

### DISCUSSION THREAD

**Discussion by recore67** [atteneder/glTFast#551] [Q&A] [answered]
is there a way to get the smoothness slider on a gltf shader, or perhaps a way to turn down the reflection scale?

**Comment by atteneder** [comment, marked as answer]
Hi,

glTF materials/shaders have a `roughness` property, which controls the same physical property as `smoothness` does on other shaders. It's just the inverse (mathematically `roughness = 1-smoothness`).

For even greater control of the specularity of a material, there's also the glTF extension KHR_materials_specular, which is not yet supported (see #208).

hth
